### DISCUSSION TITLE
YouTube Playables

### DISCUSSION THREAD

**Discussion by s0urce-c0de** [3kh0/ext-remover#1144] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
[YouTube Playables](https://www.youtube.com/playables) must be running on a vm on an OS (i'm putting my bets on android). If we can access this in school, then not only can we play games, but maybe get access to a vm. Any ideas?

**Comment by 3kh0** [comment, marked as answer]
No, they are just web games running in your browser. There is no VM running on a Google server.
